save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/OrganizationDto.kt

```diff
@@ -26,6 +27,7 @@ data class OrganizationDto(
     val avatar: String? = null,
     val description: String = "",
     val canCreateContests: Boolean = false,
+    val canBulkUpload: Boolean = false,
```

Comment:
`Bulk`?  what is the `Bulk`

may be `Vuln`?

Reply:
it is an array